connectors/sources/generic_database.py

```diff
@@ -316,6 +320,15 @@ async def _async_connect(self, query):
 
     async def close(self):
         """Close the connection to the database server."""
+        if self.dialect == "Microsoft SQL" and os.path.exists(
+            "ssl_certificate_mssql.pem"
+        ):
+            try:
+                os.remove("ssl_certificate_mssql.pem")
+            except Exception as exception:
+                logger.warning(
+                    f"Something went wrong while removing temporary certificate file. Exception: {exception}"
+                )
```

Comment:
Should this logic be in `mssql.py`?

Reply:
We checked that we need to keep the file existing while our connection is open with mssql. that's why we move this code to the `close` method.

Reply:
I understand, but you can also have `close` method in `mssql.py`

Reply:
sure we will override the close method.